src/Typechecker.ml

```diff
@@ -10,6 +10,8 @@ type error =
 
 type 'e lerror = ([> error ] as 'e) Location.located
 
+let type_to_string ty = Sexp.to_string @@ Type.sexp_of_t ty
```

Comment:
We should move this function to the pretty-printer module and reuse its function `of_type`.

Reply:
I created `Str` module in PrettyPrinter, but maybe it's redundant